src/parser.js

```diff
@@ -0,0 +1,143 @@
+const yargs = require('yargs/yargs');
+const {hideBin} = require('yargs/helpers');
+const moment = require('moment');
+
+const errorMsg = {
+  ID_LENGTH: 'ID must be a string with a length of 16',
+  DATE_FMT: 'Invalid date. Provide a date in YYYY-MM-DD HH:MM:SS format.',
+  OPTION_AMOUNT: 'At least one option is required',
+  EMPTY_REQ: 'Empty request. Please input a command',
+};
```

Comment:
- It's quite rare to see SCREAMING_SNAKE_CASE in names of object keys in JS. I would rather prefer camelCase.
- Always copying date format like this may be human error-prone, better put it in a constant.
```suggestion
const dateFmt = 'YYYY-MM-DD HH:MM:SS';

const errorMsg = {
  ID_LENGTH: 'ID must be a string with a length of 16',
  DATE_FMT: `Invalid date. Provide a date in ${dateFmt} format.`,
  OPTION_AMOUNT: 'At least one option is required',
  EMPTY_REQ: 'Empty request. Please input a command',
};
```
- The word 'amount' is used with something uncountable, so OPTION_AMOUNT identifier is a bit confusing. I would rather use optionCount/optionsAbsent